File: 2.POO/1°version/C++/Devolucion.cpp
#include <iostream>
#include <cstdlib>
using namespace std;
#include "Devolucion.h"

int main() {
    Devolucion D1;
 
    cout << "\n";
    system("color 02");
    cout << "Ingresa la fecha: ";
    getline(cin,D1.fecha);
    cout << "Ingresa el nombre: ";
    getline(cin,D1.nombre);
    cout << "Ingresa una clave de indentificacion: ";
    getline(cin,D1.clave);
    cout << "Ingresa la cantidad del prestamo: ";
    cin >> D1.cantidadprestada;
    cout << "Ingresa la cantidad devuelta: ";
    cin >> D1.cantidaddevuelta;
    cout << "Ingresa la cantidad de semanas a pagar: ";
    cin >> D1.semanaapagar;
    cout << "Ingresa el numero de la semana pagada: ";
    cin >> D1.semanapagada;
    cout << "\n";

    cout << "Nombre: " << D1.nombre << "\n";
    cout << "Fecha: " << D1.fecha << "\n";
    cout << "Clave: " << D1.clave << "\n";
    cout << "Abono: $" << D1.cantidaddevuelta << "\n";
    cout << "Semanas restantes: " << D1.semanaapagar - D1.semanapagada << "\n";
    cout << "Saldo restante: $" << D1.cantidadprestada -(D1.cantidaddevuelta * D1.semanapagada)  << "\n";
    cout << "\n";

    cout << "\n";
    cout << "------------------------------------------------------------------------------------------\n";
	cout << "Creador: Jesus A. Mex Damian 15/02/2023.";
    return 0;
}

File: 2.POO/1°version/Java/Devolucion.java
import java.util.Scanner;
public class Devolucion {
    public static void main(String[] args) {
        Scanner mex= new Scanner(System.in);

        Devoluciones D1 = new Devoluciones();
        System.out.println("\n");
        System.out.println("Ingresa la fecha: ");
        D1.fecha= mex.nextLine();
        System.out.println("Ingresa el nombre: ");
        D1.nombre= mex.nextLine(); 
        System.out.println("Ingresa la clave: ");
        D1.clave= mex.nextLine();
        System.out.println("Ingresa la cantidad prestada: ");
        D1.cantidadprestada= mex.nextDouble();
        System.out.println("Ingresa la cantidad devuelta: ");
        D1.cantidaddevuelta= mex.nextDouble();
        System.out.println("Ingresa las semanas a pagar: ");
        D1.semanaapagar= mex.nextDouble();
        System.out.println("Ingresa la semana pagada: ");
        D1.semanapagada= mex.nextDouble();
        
        
        System.out.println( "Nombre: " + D1.nombre);
        System.out.println("Fecha: " + D1.fecha);
        System.out.println("Clave: " + D1.clave);
        System.out.println("Abono: $" + D1.cantidaddevuelta);
        System.out.println("Semanas restantes: " + (D1.semanaapagar - D1.semanapagada));
        System.out.println("Saldo restante: $" + (D1.cantidadprestada -(D1.cantidaddevuelta * D1.semanapagada)));

        
        System.out.println("---------------------------------------------------------------------------");
        System.out.println("Creador: Jesus A. Mex Damian 21/02/2023.");


    }
}
